### DISCUSSION TITLE
升级到 umi4 后，使用 mfsu 启动时：JavaScript heap out of memory，有人遇到过吗 ？

### DISCUSSION THREAD

**Discussion by iaosee** [umijs/umi#10683] [Q&A]
升级到 umi4 后，开启 mfsu 启动的时候，提示内存不足，有人遇到过吗 ？

> FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory 

不开启 mfsu （mfsu: false）时能正常启动。

**Comment by fz6m** [comment]
node 版本应该太旧了，升级到 node 18 吧，或者尝试指定 `--max-old-space-size=6144`

**Reply by iaosee** [reply]
我使用 Node 16 / 18 都试过了，`--max-old-space-size` 也设置了，也是一样的情况，感觉没有什么效果。
我暂时是没开启 mfsu 。

**Reply by fz6m** [reply]
提供下设备参数情况看看。

**Reply by iaosee** [reply]
可能确实是我的内存不多了，开启 mfsu 也有些占内存，我这个内存是 8G，我看别人 16G 的没问题。
